modules/ROOT/examples/live-demos/inline/index.html

```diff
@@ -1,6 +1,6 @@
 <div class="demo-inline">
   <div class="container">
-    <p><img src="https://www.tiny.cloud/images/email-banner-2.png" style="width: 100%;"/></p>
+    <p><img src="../_images/tiny-banner.png" style="width: 100%;"/></p>
```

Comment:
Not happy with using a relative path  but that was how I could get it work on localhost